### PR TITLE
AB#85858 Application list view

### PR DESCRIPTION
**Pull request by Carq**
image

### CODE REVIEW

**Review by dawid-pawulski — APPROVED** [review]

File: src/HE.Investment.AHP.WWW/wwwroot/stylesheets/govuk-frontend-extensions.min.css [hunk 1]
```diff
@@ -50,3 +50,35 @@
 .guidance-subpart-link {
     text-decoration: underline;
 }
+
+govuk-\!-width-one-fifth {
+  width: 100% !important;
```
Comment (Antek012) [inline src/HE.Investment.AHP.WWW/wwwroot/stylesheets/govuk-frontend-extensions.min.css:55]: This is not true

File: src/HE.Investment.AHP.WWW/wwwroot/stylesheets/govuk-frontend-extensions.min.css [hunk 2]
```diff
@@ -50,3 +50,35 @@
...
+}
+
+@media (min-width: 40.0625em) {
+  .govuk-\!-width-one-fifth {
+    width: 20% !important;
+  }
+}
+
+.govuk-\!-width-one-sixth {
+    width: 100% !important;
```
Comment (Antek012) [inline src/HE.Investment.AHP.WWW/wwwroot/stylesheets/govuk-frontend-extensions.min.css:65]: also not true

File: src/HE.Investment.AHP.WWW/wwwroot/stylesheets/govuk-frontend-extensions.min.css [hunk 3]
```diff
@@ -50,3 +50,35 @@
...
+}
+
+@media (min-width: 40.0625em) {
+  .govuk-\!-width-one-sixth {
+    width: 16% !important;
+  }
+}
+
+.govuk-\!-width-one-eighth {
+  width: 100% !important;
```
Comment (Antek012) [inline src/HE.Investment.AHP.WWW/wwwroot/stylesheets/govuk-frontend-extensions.min.css:75]: again not true

File: src/HE.Investment.AHP.WWW/wwwroot/stylesheets/govuk-frontend-extensions.min.css (hunk 1 above)
Comment (Carq) [reply src/HE.Investment.AHP.WWW/wwwroot/stylesheets/govuk-frontend-extensions.min.css:55]: This is true for small displays. Then every cell should take the full width of the screen. The same approach is for gds styles.

**Pull request merged**